### PR TITLE
bug 180 - Add Showing 0 results state

### PR DESCRIPTION
**Pull request by brianelton**

### CODE REVIEW

File: src/components/product-grid/product-grid.js [hunk 1]
```diff
@@ -10,7 +10,14 @@ function init() {
   componentEl = document.querySelector('[data-component="grid"]');
 }
 function update(products){
-  componentEl.innerHTML = renderTiles({products});
+  if(products){
+    if(componentEl.style.display = 'block')
+      componentEl.style.display = 'grid';
```
Comment (aevanson) [inline src/components/product-grid/product-grid.js]: minor style comment—not a blocker—I find it a little clearer and safer to only use 'bracketless' if statements when the code all fits on one line. If the code wraps like it does here, I would recommend using brackets.

File: src/components/product-grid/product-grid.js [hunk 2]
```diff
@@ -10,7 +10,14 @@ function init() {
   componentEl = document.querySelector('[data-component="grid"]');
 }
 function update(products){
-  componentEl.innerHTML = renderTiles({products});
+  if(products){
+    if(componentEl.style.display = 'block')
```
Comment (aevanson) [inline src/components/product-grid/product-grid.js]: should be `==` or `===` here.

I'm also wondering if the check is necessary. What's the harm in just always setting the value to `grid` in this case?

Comment (brianelton) [reply src/components/product-grid/product-grid.js]: @aevanson - if the display is grid, the error message is squished into a block that is 1/3 of the width.

File: src/components/product-grid/product-grid.js (hunk 1 above)
Comment (brianelton) [reply src/components/product-grid/product-grid.js]: I have updated.

File: src/components/product-grid/product-grid.js (hunk 2 above)
Comment (brianelton) [reply src/components/product-grid/product-grid.js]: I have updated

**Pull request merged**